test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -37,6 +37,12 @@ const (
 	compressTestYaml           = "deployments/qat_dpdk_app/test-compress1/kustomization.yaml"
 	cryptoTestYaml             = "deployments/qat_dpdk_app/test-crypto1/kustomization.yaml"
 	cryptoTestGen4Yaml         = "deployments/qat_dpdk_app/test-crypto1-gen4/kustomization.yaml"
+
+	// The numbers for test below are from the document "Intel QuckAssist Technology Software for Linux*".
+	// 1 = symmetric, 2 = rsa, 4 = dsa, 8 = ecdsa, 16 = diffie-hellman, 32 = compression
+	// e.g 6 = rsa & dsa, 63 = all tests
+	symmetricTestCode   = "1"
+	compressionTestCode = "32"
```

Comment:
hardcoded strings are not so good IMO because we may also have bitmasks added together. perhaps:
```suggestion
	symmetric = 1 << iota
        rsa
        dsa
        ecdsa
        dh
        compression
```

Reply:
Ah, iota. I see.!
Thank you for the advice.! let me fix it.